QueAns/views.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponse
+from .models import Voting,Meetings,Satisfaction
+from datetime import datetime
+
+
+def upvote(request,user_id,question_id):
+    if request.method == "POST":
+        meeting = Meetings.objects.filter(question_id=question_id)
```

Comment:
- What would `.filter()` return here?
- Would it return only one object or list of objects?
- Don't you think this bug should've been caught in tests?(This is the purpose of tests so that latent bugs are minimal and every condition is tested. Now to it looks like this condition is not even reached by tests)
- Also, since you would need only one column of this whole table viz 	`end_time`, you can optimize your database call via queryset by using 	[values_list](https://docs.djangoproject.com/en/2.0/ref/models/querysets/#values-list), so how is this efficient? - Well when you fetch something normally using Django, it fetches the whole rows that match with that condition, now that table might have many columns(like this) but we are only interested in one or two columns, then we should also only fetch those columns. This speeds up our access for those rows from database.

Reply:
.filter() will return a QuerySet which will have a single object of Meeting of the given question_id.

Reply:
.filter will return objects inform  of `[obj]` instead of `obj`